packages/itwin/measure-tools/src/tools/MeasureToolDefinitions.ts

```diff
@@ -36,14 +41,14 @@ export class MeasureToolDefinitions {
     });
   }
 
-  public static getMeasureDistanceToolCommand(allowedViewportCallback: (vp: ScreenViewport) => boolean = (() => true), enableSheetMeasurements: boolean) {
+  public static getMeasureDistanceToolCommand(allowedViewportCallback: (vp: ScreenViewport) => boolean = (() => true), enableSheetMeasurements: boolean, formatting?: DistanceMeasurementFormattingProps) {
```

Comment:
can you remove the deprecated tool definitions so we can make sure users arent using the older ones which wont be using the formatting apis weve added in this pr?